shaq_overflow/app/views/answers/_vote_down.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to("Down Vote", '/', class: "up-vote-answer", data: { direction: "down", url: "/questions/#{@question.id}/answers/#{answer.id}/votes", answer_id: answer.id, question_id: @question.id, votable_type: "Answer" }) %>
```

Comment:
You might be better off making the '/' '#' or similar here. If someone clicks the link without javascript on, you probably want nothing to happen rather than then going to your root route.